scripts/contributing.js

```diff
@@ -0,0 +1,36 @@
+/* global jQuery*/
+
+(function($){
+    var githubIssuesEndpoint = 'https://api.github.com/repos/babel/babel/issues';
```

Comment:
I initially didn't include them because I wanted to know how well this would be received and I was a bit worried about sending too large of a PR.

I'll add issues for more repos over the next day over the next day or so.